addons/godot-xr-tools/user_settings/user_settings.gd

```diff
@@ -42,6 +48,8 @@ func _ready():
 func reset_to_defaults() -> void:
 	# Reset to defaults
 	snap_turning = XRTools.get_default_snap_turning()
+	y_axis_dead_zone = 0.1
+	x_axis_dead_zone = 0.2
 	player_height_adjust = 0.0
```

Comment:
it is posible to access the _define_project_setting/default value to avoid hardcode this?

Reply:
Not that I'm aware off..

Reply:
Actually, looking at this, player adjust is something specific to the user settings.

You need to look at how snap_turning works, the idea here was always that the settings in XRTools form a base set through project settings, and usersetting override this, they aren't means to be written back into the project settings.

It's the usersettings that are then used in the application.  That also removes the requirement of signals because usersettings always contain the values directly accessible.

I think you can even add a helper function `get_adjusted_vector2(p_controller, p_input_action)` to user settings that processes the deadzone and returns an adjusted vector. That way it can be more widely used than purely in the movement script.

Reply:
Ok, now its more clean solution. I was mixing project and user settings concepts. Now it kepts updated 

I kept the declaration of set_y_axis_dead_zone and set_y_axis_dead_zone but they are not used anywhere, the same as other properties